arch/powerpc/kernel/eeh_sysfs.c

```diff
@@ -91,6 +91,27 @@ static ssize_t eeh_pe_state_store(struct device *dev,
 
 static DEVICE_ATTR_RW(eeh_pe_state);
 
+#ifdef CONFIG_PCI_IOV
```

Comment:
Does it make sense to also have a eeh_notify_resume_show which would then provide the status ?

Reply:
by status do you mean the return of the rtas call? we will have to store it somewhere the return code. Also, the rtas call does not return any status of pes. I would of like that but then we have to define them individually per vf. i am up for suggestions. i can make either way. however, if we want the the status of vf pe then we need to get pfw to make changes to the rtas call to return the status.

Reply:
This one is up for debate, up to everyone else. I dont care, its just if you want to have someone in userspace do a read which will then provide some sort of status or state.